src/edu/wright/cs/raiderplanner/controller/MainController.java

```diff
@@ -99,13 +102,60 @@ public static void setSpc(StudyPlannerController newSpc) {
 	 * importing an existing Study Planner file.
 	 */
 	public static void initialise() {
-		try {
-			ui.showStartup();
-		} catch (IOException e) {
-			UiManager.reportError("Invalid file.");
-			System.exit(1);
+		File[] files = MainController.ui.getSavesFolder().listFiles();
+		if (files.length == 0 ) {
+			if (files.length == 1 && files[0].getName().contains("SamplePlanner.dat")) {
```

Comment:
In a future commit, it may be a good idea to move the string `"SamplePlanner.dat"` to a class varible so there aren't useful hard-coded strings.